pkg/apis/config/feature_flags_test.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2020 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/tektoncd/triggers/pkg/apis/config"
+	"github.com/tektoncd/triggers/test"
+)
+
+func TestNewFeatureFlagsFromConfigMap(t *testing.T) {
+	type testCase struct {
+		expectedConfig *config.FeatureFlags
+		fileName       string
+	}
+
+	testCases := []testCase{{
+		expectedConfig: &config.FeatureFlags{
+			EnableAPIFields: "stable",
```

Comment:
nit: Use the const fields from the config package.

Reply:
This was intentional -- "stable" and "alpha" are only used in expected test outputs.